vignettes/Introduction.Rmd

```diff
@@ -210,7 +225,7 @@ values[[3]]$values
 
 5) Define these variables in the query to sort out the values we want.
 
-```{r}
+```{r, comment=NA, encoding = "UTF-8"}
 
 mydata <- ApiData("https://data.ssb.no/api/v0/en/table/07964/",
                 Tid = c("2019", "2020"), # Define year to 2019 and 2020

```

Comment:
Tid = c("2021", "2022"), # Define year to 2021 and 2022

Reply:
Tar med dette når jeg sender til CRAN. Men merger som det er nå. Ellers må jeg be om ny "approve".